activity_calendar/models.py

```diff
@@ -379,7 +383,7 @@ class ActivitySlot(models.Model):
     start_date = models.DateTimeField(blank=True, null=True,
         help_text="If left empty, matches start date with activity")
     end_date = models.DateTimeField(blank=True, null=True,
-        help_text="If left empty, matches end date with activity")
+        help_text="If left empty, matches end date with activity")   
```

Comment:
NIT: please configure your editor to remove trailing spaces

https://editorconfig.org/#download